### DISCUSSION TITLE
Vercel部署的RSSHub无法使用依赖Puppeteer的路由

### DISCUSSION THREAD

**Discussion by fyihang** [DIYgod/RSSHub#11047] [Q&A] [answered]
### 路由地址

```routes
NOROUTE
```

### 完整路由地址

```fullroutes
NOROUTE
```

### 相关文档

https://docs.rsshub.app/install/#bu-shu-dao-vercel-zeit-now

### 预期是什么？

Vercel部署的RSSHub能够正常访问依赖Puppeteer的路由

### 实际发生了什么？

Vercel 部署的 RSSHub 能够无法访问到任何依赖 Puppeteer 的路由，而众所周知的是 Vercel 没有命令行窗口，我无法在部署成功后键入 `npm install` 指令。即使我在 Vercel 的 Project Settings 的 Build & Development Settings 中将 Install command 覆盖成 npm install 也会得到相同的结果。

### 部署

自建

### 部署相关信息

OS: Vercel, Node version: v16.16.0

### 额外信息

```shell
Error message: Could not find expected browser (chrome) locally. Run `npm install` to download the correct Chromium revision (1045629).
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

**Comment by TonyRL** [comment, marked as answer]
You can host browserless somewhere else and use `PUPPETEER_WS_ENDPOINT` to connect to your self-hosted puppeteer service. For Heroku, simply add [puppeteer buildpack](https://github.com/jontewks/puppeteer-heroku-buildpack).
